manifests/09_operator-ibm-cloud-managed.yaml

```diff
@@ -26,7 +26,8 @@ spec:
         seccompProfile:
           type: RuntimeDefault
       serviceAccountName: marketplace-operator
-      nodeSelector: {}
```

Comment:
@kuiwang02 @jianzhangbjz 

Could you please help us to know if we are testing the IBM pod in any place? 
Have we a test for :

```
    config.openshift.io/inject-proxy: "marketplace-operator"
    include.release.openshift.io/hypershift: "true"
    include.release.openshift.io/ibm-cloud-managed: "true"
    capability.openshift.io/name: "marketplace"
```

Reply:
The code is part of this repository
So, we need to have a test to ensure it as a pre-merge.
Not sure if I understand how that works